### ISSUE TITLE
Problem with short-duration HRRR runs

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja-mobile#83]
In some cases (maybe just short duration HRRR runs, but possibly other scenarios too) the jobs fails, although WindNinja runs properly (the WindNinja output files look good, but the json files are not created). Some error occurs at the beginning of the run: ERROR | local variable 'conversion' referenced before assignment". I'm able to run this same case on the server without errors.

See below for an example job.json for a job where this happened.

 {"status": "succeeded", "account": "nwagenbrenner@fs.fed.us", "name": "Hamilton2", "messages": ["2016-04-27T19:05:44.196641 | info | job created", "2016-04-27T13:05:45.542958-06:00 | INFO | Initializing WindNinja Run", "2016-04    -27T13:05:45.578905-06:00 | INFO | DEM created", "2016-04-27T13:06:02.592929-06:00 | INFO | WindNinjaCLI executed", "2016-04-27T13:06:02.593255-06:00 | ERROR | local variable 'conversion' referenced before assignment", "2016-04    -27T13:06:02.594032-06:00 | ERROR | local variable 'conversion' referenced before assignment", "2016-04-27T13:06:18.839872-06:00 | INFO | TopoFire tiles compiled", "2016-04-27T13:06:18.840146-06:00 | INFO | Output converted to     raster tiles", "2016-04-27T13:06:18.840830-06:00 | INFO | Complete - total processing: 0:00:33.299711"], "id": "208788c5-c746-4b1d-8e84-5a3d4c5b55d5", "output": {"products": [{"files": [], "package": "topofire.zip", "type": "ba    semap", "name": "TopoFire Basemap", "format": "tiles"}, {"files": [], "name": "WindNinja Raster Tiles", "package": "tiles.zip", "type": "raster", "format": "tiles", "data": []}]}, "input": {"domain": {"xmin": -114.3671141099184    5, "ymin": 46.1468046488759, "ymax": 46.31239546640845, "xmax": -114.09856979269532}, "products": "vector:true;raster:true;topofire:true;geopdf:false;weather:true", "parameters": "forecast_duration:1;vegetation:grass;mesh_choic    e:fine", "forecast": "NOMADS-HRRR-CONUS-3-KM"}, "email": ""}

log.txt:

2016-04-27T19:05:45.541995 | INFO | Begin - version 2016.03.01.11
  2 2016-04-27T19:06:02.593172 | INFO | ERROR       local variable 'conversion' referenced before assignment
  3 2016-04-27T19:06:02.593218 | INFO | ERROR       local variable 'conversion' referenced before assignment
  4 2016-04-27T19:06:18.841357 | INFO | Complete - total processing: 0:00:33.299711
  5 2016-04-27T19:06:18.841865 | INFO | Job dueued

**Comment by fspataro** [comment]
can you zip up the job directory and send it me.  I'll run the job here and see what that error is about.  It was expecting something in the windninja output that it didn't find.

**Comment by fspataro** [comment]
This appears to be an issue with the "past" filter.  None of the results were 'current or in the future' so the 'conversion' dataset variable was never set to before being used.  

1. This should never occur now that we've remove the "past" filter 
2. I changed the loop a little so it doesn't have to check if "conversion" is valid at the end.
3. I added an error check for empty array of shapefiles at the end of the CLI method. 

This should clean it up for most the possible scenarios.
